fs_src/script.js

```diff
@@ -815,16 +812,17 @@ function sendMessageWebSocket(method, params = [], id = requestID) {
   requestID += 1
   return new Promise(function (resolve, reject) {
     try {
+      console.log("[send] " + method + " " + id + ":", params);
       socket.send(JSON.stringify({"method": method, "id": id, "params": params}));
-      console.log("[send] Data sent: " + JSON.stringify({"method": method, "id": id, "params": params}));
+      console.log("[sent] " + method + " " + id);
```

Comment:
missing `params` ?

Reply:
as this is only the done message with the same id as the log from 2 lines above I don't want to repeat it

Reply:
ok